Dockerfile

```diff
@@ -1,6 +1,6 @@
 # syntax=docker/dockerfile:1
 
-FROM ghcr.io/imagegenius/baseimage-ubuntu:lunar
+FROM ghcr.io/imagegenius/baseimage-immich:latest
```

Comment:
Wouldn't it better to keep the release name such as `ghcr.io/imagegenius/baseimage-immich:lunar`